src/uct/ib/mlx5/dv/ib_mlx5_dv.c

```diff
@@ -462,6 +462,18 @@ ucs_status_t uct_ib_mlx5_devx_query_ooo_sl_mask(uct_ib_mlx5_md_t *md,
     return UCS_OK;
 }
 
+void uct_ib_mlx5_devx_set_qpc_dp_ordering(void *qpc,
+                                          const uct_ib_iface_t *iface)
```

Comment:
maybe pass iface->config.dp_ordering_ooo instead of iface object?

Reply:
fixed